syncer/syncer.go

```diff
@@ -612,18 +614,17 @@ func (s *Syncer) getTable(origSchema, origTable, renamedSchema, renamedTable str
 func (s *Syncer) trackTableInfoFromDownstream(origSchema, origTable, renamedSchema, renamedTable string) error {
 	// TODO: Switch to use the HTTP interface to retrieve the TableInfo directly
```

Comment:
going to fix this TODO in another PR

Reply:
Now, we have TLS support, if switching to HTTP API, do we need to handle it, right?